sensor_msgs/PointCloud2 cloud
---
float32 a
float32 b
float32 c
float32 d